components/post_view/post_list_virtualized/post_list_virtualized.jsx

```diff
@@ -126,7 +126,7 @@ export default class PostList extends React.PureComponent {
 
         window.addEventListener('resize', this.handleWindowResize);
 
-        EventEmitter.addListener('scroll_post_list_to_bottom', this.scrollToBottom);
+        EventEmitter.addListener(EventTypes.POST_LIST_SCROLL_CHANGE, this.scrollToBottom);
```

Comment:
Do these need to be changed to use `this.scrollChange`?

Reply:
Nice catch. Fixed it.